### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Disk_UploadVHD_ScreenReader: Voiceover reads the browse button as "No file Selected" which is incorrect.

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/AzureStorageExplorer#2068]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #VoiceOver; #Mac;

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.1
MAC Version 10.14.6 Beta(18G29g)

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Disks" of the selected subscription.
5. Navigate to "A11ytest" and  click on it.
6. Navigate to upload button and click on it.
7. "Upload VHD" Popup gets opened.
8.Start VoiceOver and using Tab key and check whether the Voiceover reads the browse button as "No file Selected".
**​Actual:​**
Voiceover reads the browse button as "No file Selected" which is incorrect.
**Expected:**
VoiceOver should read as browse button .
**UserImpact:**
Screen reader user unable to know the actual behavior of the button.
Recommendation:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference:**
MAS1.3.1:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
**Attachment for Reference:**
[2068_A11y_AzureToolsStorageExplorer_Disk_UploadVHD_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3720659/2068_A11y_AzureToolsStorageExplorer_Disk_UploadVHD_ScreenReader.pptx)

**Comment by MRayermannMSFT** [comment]
Fix merged into master.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by nirmalsahoo** [comment]
@MRayermannMSFT Verified on version 1.12.0 and MAC 10.15.3.The issue is no longer reproduced.hence close this issue.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;
